portal/src/main/scala/cromwell/pipeline/datastorage/dto/users/UserResponse.scala

```diff
@@ -0,0 +1,11 @@
+package cromwell.pipeline.datastorage.dto.user
+
+import cromwell.pipeline.datastorage.dto.{User}
+import play.api.libs.json.{Json, OFormat}
+
+final case class UserResponse(user: User)
+
+object UserResponse {
+  implicit val userFormat: OFormat[User] = Json.format[User]
```

Comment:
`User` formatter does not belong here. It has nothing to do with `UserResponse` class.

Reply:
done